src/main/java/frc/robot/subsystems/Ting.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Ting extends SubsystemBase {
+    CANSparkMax motor;
+    CANSparkMax moto2;
+    SingleJointedArmSim arm;
+    public Mechanism2d ting1;
+    MechanismLigament2d ligma;
```

Comment:
All instance variables should be private. To access ting1, use a get method. Also, you only have one motor, so you can remove motor2. They put a Falcon 500 on the robot, so you should change this to a TalonFX as well.